rero_ils/modules/loans/api.py

```diff
@@ -140,6 +145,21 @@ def pid(self):
         """Shortcut for pid."""
         return self.get('pid')
 
+    @property
+    def rank(self):
```

Comment:
This shortcut is only available for loans generated in the method `patron_profile`, i think you can remove it because it does not apply for all loans

Reply:
This shortcut is used by the function `sorted`